src/stereotype_client.js

```diff
@@ -105,6 +105,10 @@ class StereotypeClient {
     this.curieHeader = String(headerValue);
   }
 
+  setMaximumCrawlDepthHeader(headerValue) {
```

Comment:
It would be nice if there was a way to pass all options in one go.

Reply:
Would you prefer me to do this in this PR or a separate one?